### DISCUSSION TITLE
How to use mysql as my source when I use pnpm start:backend in Development Setup

### DISCUSSION THREAD

**Discussion by da-da-da** [nocodb/nocodb#7068] [Q&A] [answered]
When I use like `pnpm start:backend --e NC_DB="mysql2://host.docker.internal:3306?u=root&p=password&d=d1" -e NC_AUTH_JWT_SECRET="569a1821-0a93-45e8-87ab-eb857f20a010"` to connect mysql, the system use default sqlite.

**Comment by wingkwong** [comment, marked as answer]
try this instead: 

`NC_DB="mysql2://host.docker.internal:3306?u=root&p=password&d=d1" NC_AUTH_JWT_SECRET="569a1821-0a93-45e8-87ab-eb857f20a010" pnpm start:backend`
